config/rbac/role.yaml

```diff
@@ -67,6 +67,7 @@ rules:
   - pods/status
   verbs:
   - get
+  - patch
```

Comment:
[0] Just want to make sure I understand how this stuff works:

This is a generated file, right?  Or is this one maintained by hand?

And this line was added because of the patch call made [here](https://github.com/apache/solr-operator/pull/530/files#diff-9f5886fd88f144a700c67988483eaba63bacab421a67dbf78adbd734bcbbdd00R126)?  Really cool that the generator is set up to that!

Reply:
This is a generated file, and it would be really cool if it was automatically added based on what commands you use... But alas its a little less cool.

This is what determines this (each controller has a separate list, that all get combined to make this file): https://github.com/apache/solr-operator/pull/530/files#diff-27cb619f80d28ac158f7885b7925d8582babfa7fe1f824599d65a745302d237cR67